x/stakeibc/keeper/icqcallbacks_withdrawal_reward_balance.go

```diff
@@ -54,14 +58,34 @@ func WithdrawalRewardBalanceCallback(k Keeper, ctx sdk.Context, args []byte, que
 	k.Logger(ctx).Info(utils.LogICQCallbackWithHostZone(chainId, ICQCallbackID_WithdrawalRewardBalance,
 		"Query response - Withdrawal Reward Balance: %v %s", withdrawalRewardBalanceAmount, tradeRoute.RewardDenomOnHostZone))
 
-	// Using ICA commands on the withdrawal address, transfer the found reward tokens from the host zone to the trade zone
-	if err := k.TransferRewardTokensHostToTrade(ctx, withdrawalRewardBalanceAmount, tradeRoute); err != nil {
+	// Split the withdrawal amount into a rebate, stride fee, and reinvest portion
+	feeInfo, err := k.CheckForCommunityPoolRebate(ctx, chainId, withdrawalRewardBalanceAmount)
+	if err != nil {
+		return errorsmod.Wrapf(err, "unable to check for community pool rebate")
+	}
+
+	// If there's a rebate portion, fund the community pool with that amount
+	if feeInfo.RebateAmount.GT(sdkmath.ZeroInt()) {
+		rebateToken := sdk.NewCoin(tradeRouteCallback.RewardDenom, feeInfo.RebateAmount)
+		if err := k.FundCommunityPool(ctx, hostZone, rebateToken, types.ICAAccountType_WITHDRAWAL); err != nil {
```

Comment:
Reminder, also pass in the Withdrawal ICA address as Depositor on the actual MsgFundCommunityPool

Reply:
handled it by switching on the account type